recipe/build.sh

```diff
@@ -1,32 +1,42 @@
 #!/usr/bin/env bash
-# Get an updated config.sub and config.guess
-cp $BUILD_PREFIX/share/gnuconfig/config.* ./CoinUtils
```

Comment:
If you keep these lines, do you still need the extra flags for osx-arm64? The updated config.* files usually help.

Reply:
Good idea. Will try soon.